### ISSUE TITLE
Normalize training data

### ISSUE THREAD

**Issue by nya-elimu** [elimu-ai/ml-storybook-reading-level#28]
https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.normalize.html

**Comment by howdyDp** [comment]
I was trying to normalize the data, i wanted to know what kind of normalization are you looking for ? 
min max normalization or z-scaled normalization and this is my first open source contribution so if you have any advice do let me know

**Comment by jo-elimu** [comment]
@howdyDp We are looking for whichever normalization technique that would result in an improved [accuracy score](pmml/step3_predict/step3_2_accuracy_score.txt).

> [!TIP]
> Try running `python run_all_steps.py` for each kind of normalization you want to try, and see of that results in improvements in `step3_2_accuracy_score.txt` and `step3_2_mean_absolute_error.txt`

And if it turns out that normalization does _not_ improve the accuracy, then we wouldn't use normalization at all.

**Comment by eve-b612** [comment]
Hi! This is my first contribution, I've normalised the data 3 ways and wanted to share the results. My work is in a jupyter notebook, I've forked the repo and created a new branch named 'normalisation-experiment'. I wanted to know which folder to upload this in the repo branch?

**Comment by eve-b612** [comment]
Sent the pull request for review! :)

**Comment by alaettin10** [comment]
Hi! I'd like to work on this issue. I’m thinking of trying a few normalization methods and comparing their accuracy and MAE with the current results. Is this issue still relevant?

**Comment by jo-elimu** [comment]
@alaettin10 Yes, please give it a try 🙂

Just be aware that we have a [GitHub Action](https://github.com/elimu-ai/ml-storybook-reading-level/actions/workflows/pmml-nightly.yml) that is running once per day, which involves triggering a Python script. So the recommended technique for implementation would be as a Python file and not as a notebook.
